### DISCUSSION TITLE
Welcome to AzureNamingTool Discussions!

### DISCUSSION THREAD

**Discussion by BryanSoltis** [mspnp/AzureNamingTool#75] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by epfaffinger** [comment]
Good morning @BryanSoltis , I have been running v4.0.0 for a while now and want to thank you for a great tool. It's ideal to ensure Azure resource naming is consistent. I tried to upgrade to 4.1.0 but although the workflow runs successfully and without error it appears to have wiped out my App Service. All I get now is the default "Your web app is running and waiting for your content" landing page.

I was wondering if there is anyway to push or rev3ery back to earlier code? I submitted a bug report but that was probably not the best place to post?
